cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/tests/tests.rs

```diff
@@ -413,25 +413,6 @@ pub fn can_calculate_fee_for_standalone_message_confirmation_transaction() {
 	)
 }
 
-#[test]
-fn open_and_close_bridge_works() {
-	let origins = [SiblingParachainLocation::get(), SiblingSystemParachainLocation::get()];
-
-	for origin in origins {
-		bridge_hub_test_utils::test_cases::open_and_close_bridge_works::<
-			Runtime,
-			XcmOverBridgeHubRococoInstance,
-			LocationToAccountId,
-			WestendLocation,
-		>(
-			collator_session_keys(),
-			bp_bridge_hub_westend::BRIDGE_HUB_WESTEND_PARACHAIN_ID,
-			origin,
-			BridgedUniversalLocation::get(),
-		)
-	}
-}
-
```

Comment:
why remove this unrelated test? 😕

Reply:
When I pulled the latest changes, GitHub was highlighting me that test with green. (from Files Changed tab)
I thought that the test was removed meanwhile and decided to remove it from my branch as well. 
Even now, when I open "Files Changed" tab it shows me that I only added location conversion test. 
I thought "Files Changed" tab was commit agnostic, wasn't it?

Reply:
https://github.com/paritytech/polkadot-sdk/commit/710e74ddefdff1e36b77ba65abe54feb0ac15040